tests/test_browse.py

```diff
@@ -59,6 +63,13 @@ def test_abs_with_truncated_author_list(self):
         assert b'additional authors not shown' in rv.data, \
             'abs/1411.4413 should have a truncate author list'
 
-
-if __name__ == '__main__':
-    unittest.main()
+    def test_all_abs_as_web_pages(self):
+        num_files_tested = 0
+        for dir_name, subdir_list, file_list in os.walk(ABS_FILES):
+            for fname in file_list:
+                fname_path = os.path.join(dir_name, fname)
+                if os.stat(fname_path).st_size == 0 or not fname_path.endswith('.abs'):
+                    continue
+                m = AbsMetaSession.parse_abs_file(filename=fname_path)
+                rv = self.app.get(f'/abs/{m.arxiv_id}')
+                self.assertEqual(rv.status_code, 200)
```

Comment:
Nice test! Other than checking for status 200, would it be worth checking for particular bits of common content?